### ISSUE TITLE
Update release procedure

### ISSUE THREAD

**Issue by saig0** [camunda/feel-scala#732]
## Description

Currently, the release procedure is documented in the "Release" issue template. Since the release switched from Jenkin to GitHub actions, the procedure is not correct anymore.

Update the procedure based on the notes of the `1.17.0` release.

### Notes 

- Create a new branch `release-1.17.0` from `main`
- Push branch
- Open release workflow https://github.com/camunda/feel-scala/actions/workflows/release.yml
- Run workflow with:
	- from branch: `main`
	- release branch: `release-1.17.0`
	- release version: `1.17.0`
	- next version: `1.18.0-SNAPSHOT`
- Wait until workflow is finished
- Release the version in Nexus: https://oss.sonatype.org/#stagingRepositories
- Go to https://github.com/camunda/feel-scala/releases and edit the draft release
	- Generate release notes
	- Remove doc and other not code relevant PRs
	- Rephrase PR titles 
	- Sort PRs by `feat`, `fix`, `chore`, `build` + dependencies
	- Publish the release
- (if major/minor release) append the [changelog in the documentation](https://camunda.github.io/feel-scala/docs/changelog/) for the released version 
	- mention the new features
	- commit the change
-  (if major/minor release) archive the documentation of the previous version
	- Run `npm run docusaurus docs:version 1.17`
	- Update the versions in the configuration `/docs/docusaurus.config.js`
	    - set the latest version in `docs > lastVersion`
	    - set the current version in `docs > versions > current > label`
	    - set the supported versions in `docs > versions` (enable/disable the "unmaintained version" banner)
- Update the version that is used by the FEEL REPL script `/feel-repl.sc` under `import $ivy.org.camunda.feel:feel-engine:1.17.0`
- Open a PR and assign a reviewer
- After the PR is merged, create a new branch `1.17`

**Comment by saig0** [comment]
Read [here](https://medium.com/@zelldon91/connecting-camunda-platform-8-with-github-workflows-ee1f91488ad3) on how to trigger a GitHub workflow from a Camunda process.

**Comment by saig0** [comment]
I modeled the new release process as a BPMN process [here](https://modeler.cloud.camunda.io/diagrams/9e3b59f9-1ada-4735-8921-526f1a24cef8--build-a-feel-scala-release?v=366,267,1). It is deployed in Camunda SaaS.

I tested the process for the release `1.17.7` and [it worked](https://bru-2.operate.camunda.io/689a796f-7efa-487f-9e44-76ca3a98c77b/processes/2251799850289005). :heavy_check_mark: 

The release process can be started via Tasklist. I saved the link in the `#ask-dmn-feel` Slack channel. :rocket:
